src/main/play-doc/operation/ContinuousDeliverySetupOps.md

```diff
@@ -0,0 +1,134 @@
+```
+The Continuous Delivery service will form a cluster among its instances, and the deployer which is responsible for deployment is sharded by bundle name and its compatible version.
+
+Once the Continuous Delivery service has been started, it will be exposed via the proxy on port `9000` under the path `/deployments`. Ensure external access is available to this port and path to allow Continuous Delivery service to be invoked by Bintray webook.
+
+## Bintray webhook setup
+
+Execute the following command to create Bintray webhook.
+
+```
+curl -v \
+     -u ${BINTRAY_USERNAME}:${BINTRAY_API_KEY} \
+     -X POST \
+     -H "Content-Type: application/json" \
+     -d '{"url": "${CALLBACK_URL}", "method": "post"}' \
+     "https://api.bintray.com/webhooks/${BINTRAY_SUBJECT}/${BINTRAY_REPO}/${BINTRAY_PACKAGE_NAME}"
+```
+
+|Term|Description|Example|
+|----|-----------|-------|
+|`BINTRAY_USERNAME`|The Bintray username. This can be obtained by signing into Bintray and viewing the user profile.|&nbsp;|
+|`BINTRAY_API_KEY`|The API key of the user. This can be obtained by signing into Bintray and viewing the API key within the user profile.|`bt-secret` is used for our example.|
+|`CALLBACK_URL`|The callback URL which will be invoked by the webhook.|&nbsp;|
+|`BINTRAY_SUBJECT`|The Bintray subject who owns the `BINTRAY_REPO` in question.|`typesafe` is an example of Bintray subject which is accessible on [https://bintray.com/typesafe](https://bintray.com/typesafe).|
+|`BINTRAY_REPO`|The Bintray repository where the bundle to be deployed resides.|`bundle` is an example of Bintray repo which is accessible on [https://bintray.com/typesafe/bundle](https://bintray.com/typesafe/bundle).<br>The repo owns various bundles such as `cassandra` which is accessible from [https://bintray.com/typesafe/bundle/cassandra](https://bintray.com/typesafe/bundle/cassandra).|
+|`BINTRAY_PACKAGE_NAME`|The Bintray package of the bundle to be deployed.|`cassandra` is an example of Bintray package which is accessible on [https://bintray.com/typesafe/bundle/cassandra](https://bintray.com/typesafe/bundle/cassandra).|
+
+
+Format the `CALLBACK_URL` as such.
+
+```
+${CD_BASE_URL}/deployments/${BINTRAY_SUBJECT}/${BINTRAY_REPO}/<subject>
+```
+
+|Term|Description|Example|
+|----|-----------|-------|
+|`CD_BASE_URL`|The URL where Continuous Delivery is exposed. This will normally be a load balancer which is mapped to proxy on port `9000` under the path `/deployments`.|`http://staging.acme.com`|
+|`BINTRAY_SUBJECT`|Must match the `BINTRAY_SUBJECT` posted to `api.bintray.com`|&nbsp;|
+|`BINTRAY_REPO`|Must match the `BINTRAY_REPO` posted to `api.bintray.com`|&nbsp;|
+
+Further details on the bintray webhook setup instructions can be found on the [https://bintray.com/docs/api/#_register_a_webhook](https://bintray.com/docs/api/#_register_a_webhook).
+
+## Manual deployment
+
+The `conduct deploy` command can be used to deploy a bundle manually through the Continuous Delivery pipeline.
```

Comment:
We should first introduce this command by describing why it is useful i.e. to manually roll forward/back through deployments.